cmd/calyptia/create_core_instance_gcp.go

```diff
@@ -138,11 +149,26 @@ func newCmdCreateCoreInstanceOnGCP(config *config, client gcp.Client) *cobra.Com
 	fs.StringVar(&sshUser, "ssh-user", "", "GCP SSH user to use for the instance. (default is no user)")
 	fs.StringVar(&sshKeyPath, "ssh-key", "", "SSH Key path to use for the instance. (default is no key)")
 	fs.StringVar(&network, "network", "default", "GCP Network")
-	fs.BoolVar(&useTestImages, "use-test-images", false, "Use GCP test images instead of released channel (only for testing/development).")
```

Comment:
Does this mean we cannot use these images now? Or we enable via the env var? It's not clear in the PR description.

I'd like to know how I use test images so what do I have to do?